### PR TITLE
Lupian Merc Update

### PR DESCRIPTION
**Pull request by Dedwolf**
## About The Pull Request

Open for feedback.

Adds a new mercenary to the game. The ~~Black Star~~ ~~Silver Dawn~~ Forlorn Hope mercenary. With Vakra swept up in civil wars and the might of the Amarir fractured, soldiers are left with splintered leadership and oft nobody to follow, but the banner of the Forlorn Hope welcomes them. 

They were once a holy order, dedicated to the Ten and sworn to Noc, now they welcome the wounded soldiers and homeless peasants as mercenaries of faith and fortune. With so many taking up the banner, and their already strained infrastructure spread thin, one can find these mercenaries wearing the tatters of civil wars, huddled in the woods in camps, swearing their service under dubious claims of righteous fervor and honor. (Some are original members of the Holy Order. Some are peasants wanting for shelter and food for their dying children. Some are veteran soldiers, or those left behind, and know nothing other than a life of soldiering.)

Spritework is courtesy of Bimmer (brigandine and volf helm), and Cre (warhammer, heater shield and heraldry)

They also have their own combat music, give it a listen.

https://github.com/user-attachments/assets/081f309a-2781-4111-b3a5-5e924d8832a3

## Why It's Good For The Game

In my opinion mercenaries being restricted based on race and lore is a GOOD thing. The main goal of this PR is to further expand on the existing lore and to offer another side to mercenaries. With this PR your mercenary can be explicitly religious, they could be a zealot, or they could be a peasant with nowhere to turn to, or they could be a soldier far from home who doesn't care for their religion and just needs a job to keep them from the disgrace of banditry.

### CODE REVIEW

**Comment by kepesk1** [comment]
Music dope as fuck and I am glad someone is finally adding brigandine medium armor later on

**Comment by Jedrockin3** [comment]
These guys seem to have way less identity than the other mercenary groups, if it's a former holy order it should be played up, Black star seems like a bad name and doesn't seem to really convey what this companies vibe is.

**Comment by Hmeister-real** [comment]
we have too many mercs, this is cool and all, but it's bloat

**Comment by katsukai** [comment]
Bad song, bad name legit sounds like a graggar company or wannabe modern company scifi merc vibes, nonsensical lore. Should sticked with cuman idea.

**Comment by radarseas** [comment]
what do these mercs look like, any chance of some screenshots of the style?

**Comment by katsukai** [comment]
Like the sprites, but I am against it being a sabre buckler mercenary due it being racially slow -1SPD and -2 PER makes it bad for aiming double accuracy areas.

**Comment by katsukai** [comment]
Play tested this and I have to say it's quite well balanced, like the warhammer addition and it being able to use as a pickaxe is cool.
Like the mercenaries being tied to a class system now. Only gripe is the mercenary name.

**Comment by pilotmans** [comment]
Forlorn Hope is a Mouthful and any substitutes are also odd like "Bereft Hope"

**Comment by Lord-Preston** [comment]
If you liked Silver Dawn I'd say go for it regardless of the Red Dawn from Lifeweb. Or if you really didn't want that similarity maybe change Dawn to Crescent which is a little more moon themed, so Silver Crescent.

Forlon Hope isn't too bad either, it could grow on me.

File: code/game/objects/items/rogueweapons/melee/blunt.dm [hunk 1]
```diff
@@ -99,6 +99,37 @@
...
+	desc = "A fine steel warhammer, makes a satisfying sound when paired with a knight's helm."
+	icon_state = "swarhammer"
+	smeltresult = /obj/item/ingot/steel
+	wdefense = 4
+
+/obj/item/rogueweapon/mace/warhammer/getonmobprop(tag)
+	if(tag)
+		switch(tag)
+			if("gen")
+				return list("shrink" = 0.5,"sx" = -9,"sy" = -8,"nx" = 9,"ny" = -7,"wx" = -7,"wy" = -8,"ex" = 3,"ey" = -8,"northabove" = 0,"southabove" = 1,"eastabove" = 1,"westabove" = 0,"nturn" = 90,"sturn" = -90,"wturn" = -90,"eturn" = 90,"nflip" = 0,"sflip" = 8,"wflip" = 8,"eflip" = 0)
```
Comment (Cyprex) [inline code/game/objects/items/rogueweapons/melee/blunt.dm:127]: The heck is this?

**Pull request merged**